.github/workflows/ci.yml

```diff
@@ -15,6 +15,15 @@ jobs:
               run: |
                   composer install
                   composer test
+    lint-api-tests:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: lint
+              run: |
+                  touch .api-testing.config.json
```

Comment:
Hm, I feel like it would make more sense to resolve the `node/no-missing-require` error with an `eslint-disable-line` directive, so that running the linter also works locally. without creating that config file.

Reply:
yup sounds good!